javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -60,11 +60,11 @@ function AppNavbar() {
           <Nav.Link>About</Nav.Link>
         </LinkContainer>
           { (isAdmin || isMember) &&
-          <NavbarHover title="Analyze Message Data" items={DataPages} />
+              <NavbarHover title="Analyze Slack Data" items={DataPages} />
           }
           { (isMember || isAdmin) && 
-              <LinkContainer to={"/member/search"}>
-                  <Nav.Link>Search</Nav.Link>
+              <LinkContainer to={"/member/GoogleSearch"}>
```

Comment:
I wonder whether this links to the correct place now?

The links are defined in `app.js`